src/uni_modules/wot-design-uni/components/wd-message-box/wd-message-box.vue

```
@@ -12,6 +12,7 @@
       :root-portal="rootPortal"
     >
       <view :class="rootClass">
+        <view class="wd-message-box__close" v-if="messageState.showClose" @click="toggleModal('close')"><wd-icon name="close" size="14px"></wd-icon></view>
```

ESLint

[error] Replace `<wd-icon·name="close"·size="14px"></wd-icon>` with `⏎··········<wd-icon·name="close"·size="14px"></wd-icon>⏎········`

(prettier/prettier)